lib/ex_webrtc/dtls_transport.ex

```diff
@@ -42,19 +42,18 @@ defmodule ExWebRTC.DTLSTransport do
   end
 
   @impl true
-  def init([ice_config, peer_connection]) do
+  def init([ice_config, ice_module, owner]) do
     # temporary hack to generate certs
-    {:ok, cert_client} = ExDTLS.start_link(client_mode: true, dtls_srtp: true)
-    {:ok, cert} = ExDTLS.get_cert(cert_client)
-    {:ok, pkey} = ExDTLS.get_pkey(cert_client)
-    {:ok, fingerprint} = ExDTLS.get_cert_fingerprint(cert_client)
-    :ok = ExDTLS.stop(cert_client)
+    dtls = ExDTLS.init(client_mode: true, dtls_srtp: true)
+    cert = ExDTLS.get_cert(dtls)
+    pkey = ExDTLS.get_pkey(dtls)
+    fingerprint = ExDTLS.get_cert_fingerprint(dtls)
 
-    {:ok, ice_agent} = ICEAgent.start_link(:controlled, ice_config)
+    {:ok, ice_agent} = ice_module.start_link(:controlled, ice_config)
```

Comment:
To be perfect we should define a behavior for `ice_module`

Reply:
Maybe let's let it be for now, especially considering that `DTLSTransport` uses exactly one function from `ICEAgent` API.